media/js/firefox/australis/browser-tour.js

```diff
@@ -504,8 +524,13 @@ if (typeof Mozilla == 'undefined') {
             buttons,
             options
         );
-
-        gaTrack(['_trackEvent', 'Tour Interaction', 'Later', 'The new Forget Button']);
+        
+        window.dataLayer = window.dataLayer || [];
```

Comment:
Is this line needed here still? (none of the other `dataLayer` calls in this file have it)